### ISSUE TITLE
Change border radius of skeleton 

### ISSUE THREAD

**Issue by vishal-akg** [dvtng/react-loading-skeleton#32]
How can i change border radius of skeleton?
No proper documentation on how can use custom style.

**Comment by vemundeldegard** [comment]
I am also interested in this.

**Comment by davidkhierl** [comment]
```
import Skeleton from 'react-loading-skeleton';

export const SkeletonLoading = styled(Skeleton)`
  span {
    border-radius: 15px !important;
  }
`;
```

Tried to do this but seems not working

**Comment by timiscoding** [comment]
I don't think that works because react-loading-skeleton doesn't pass the className generated by styled-components you're using through to the Skeleton component.

[Working code sandbox example](https://codesandbox.io/s/change-react-loading-skeleton-css-xqyg0)
